templates/html_reports/html_template.html

```diff
@@ -0,0 +1,67 @@
+    <html>
```

Comment:
Suggest you to change this to using jinja2
https://chatgpt.com/share/672a1e22-3df8-800b-af39-565991da77e1

Reply:
@petr-balogh https://github.com/red-hat-storage/noobaa-sa-ci/pull/42